apps/meteor/app/lib/server/functions/saveUser.js

```diff
@@ -421,6 +421,11 @@ export const saveUser = async function (userId, userData) {
 		updateUser.$set['emails.0.verified'] = userData.verified;
 	}
 
+	if (!updateUser.$set.customFields) {
+		updateUser.$set.customFields = {};
+	}
+	updateUser.$set.customFields.messageToAgents = {};
```

Comment:
do we also want to create the messageToAgents property when we create a new user? or saveUser is fine?

Reply:
if i understand it correctly, we want to set the messageToAgents to empty object when saveUser is invoked? why do we want to set to {} everytime the saveUser is called?

Reply:
I removed them. We might do this on the CoWorkBot side.